.github/workflows/ci-workflow.yml

```
@@ -75,6 +75,15 @@
       - name: Install dependencies
         run: yarn install
 
+      - name: Set Sentry Auth Token
+        run: echo "SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }}" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:62: Double quote to prevent globbing and word splitting

(shellcheck)